compile.sh

```diff
@@ -31,6 +31,12 @@ FORCE_CHECKOUT="yes"			# ignore manual changes to source
 BUILD_ALL="no"				# cycle through available boards and make images or kernel/u-boot packages.
 					# set KERNEL_ONLY to "yes" or "no" to build all packages/all images
 
+PRIVATE_CCACHE="no"			# Use separate ccache directory located in build tree ($DEST/ccache)
+					# Can be used to avoid problems with ownershipt on $HOME/.ccache. 
+					# Set to "yes" if you are using build script as non-root user with "sudo"
+					# note: to use a ramdisk for ccache, create first a symlink to $DEST/ccache
+                                        # Set to "no" check if $DEST/ccache exists and delete (directory/symlink)
```

Comment:
not exactly, the default config for ccache is $HOME/.ccache (here it is PRIVATE_CCACHE="no")	
with PRIVATE_CCACHE="yes", you can have a ccache folder limited to Armbian tool usage.

Armbian tool creates "root" owned files
I was in trouble ccache compiling other software "non-root" so I guess the Armbian tool should clean its own mess 

Moreover, (with more changes) it would offer the possibility to create ccache per SoC for mass compilation